lib/knetwork/BUILD.bazel

```diff
@@ -1,8 +1,22 @@
-load("@io_bazel_rules_go//go:def.bzl", "go_library")
+load("@io_bazel_rules_go//go:def.bzl", "go_library", "go_test")
```

Comment:
meta: could you move this under a subdirectory of knetwork? like knetwork dns?

rationale: knetwork right now is pulled in by the tunnel CLI interface. No reason to bring in a build time dependency on a DNS server library?

Reply:
Moved to package kdns